Directory.Build.props

```diff
@@ -40,6 +42,8 @@
   <PropertyGroup>
     <IsTest>false</IsTest>
     <IsTest Condition="$(MSBuildProjectName.ToLowerInvariant().Contains('test'))">true</IsTest>
+    <IsVsix>false</IsVsix>
```

Comment:
For ApiPort.Vsix, it sets a property that APiScan uses too, it is "<IsPackage>" on line 49.

Reply:
That's better.  Less changes. Fixed.